client/notebooks/Demo_Client_Notebook.ipynb

```diff
@@ -147,10 +147,24 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 32,
+   "execution_count": 1,
```

Comment:
There is an import error at the beginning and then the of the cells are empty
https://github.com/dscc-admin-ch/lomas/blob/wip_369_doc_to_models/client/notebooks/Demo_Client_Notebook.ipynb

Reply:
Fixed it, there were a few issues with wrong import statements.